protected/humhub/modules/admin/controllers/GroupController.php

```diff
@@ -276,13 +277,26 @@ public function actionAdminUserSearch($keyword, $id)
 
     public function checkGroupAccess($group)
     {
-        if(!$group) {
+        if (!$group) {
             throw new HttpException(404, Yii::t('AdminModule.user', 'Group not found!'));
         }
 
-        if($group->is_admin_group && !Yii::$app->user->isAdmin()) {
+        if ($group->is_admin_group && !Yii::$app->user->isAdmin()) {
             throw new HttpException(403);
         }
     }
 
+    public function actionReassignAll($id)
+    {
+        $job = new ReassignAllDefaultSpaces();
+        if (\humhub\modules\queue\helpers\QueueHelper::isQueued($job)) {
+            print "Reassign process is already queued or running!\n";
```

Comment:
Can we find a better output solution here? e.g. Rebuild search index job?

Reply:
Done!